### PR TITLE
Fix sampling of normal distribution

### PR DESCRIPTION
**Pull request by paulromano**
This PR is a replacement of #1670 since @makeclean was too lazy ("busy" I'm sure he would say :laughing:) to respond to my original comments.

### CODE REVIEW

File: tests/unit_tests/test_math.py [hunk 1]
```diff
@@ -213,13 +214,19 @@ def test_normal_dist():
 
     assert ref_val == pytest.approx(test_val)
 
-    prn_seed = 1
-    a = 14.08
+    # make sigma = 1.0
```
Comment (pshriwise) [inline tests/unit_tests/test_math.py:217]: Really small thing, but perhaps we could remove the comment here and update the variable names of `a` and `b` to `mean` and `sigma` respectively.

**Review by pshriwise — CHANGES_REQUESTED** [review]
Tried this out myself like @gridley and it seems to work quite well. Just one super small comment in the test and it's good to go. I'll close #1670 after merging this.

File: tests/unit_tests/test_math.py (hunk 1 above)
Comment (paulromano) [reply tests/unit_tests/test_math.py:217]: Good point -- I'll actually change to `mean` and `stdev`

**Review by pshriwise — APPROVED** [review]
Thanks @paulromano!

**Pull request merged**